dspace/config/dspace.cfg

```diff
@@ -331,6 +331,10 @@ doi.localpart.testsuffix = FK2dryad.
 # For dev and testing, don't want to hit them all the time
 doi.datacite.connected = ${default.doi.datacite.connected}
 
+##### Connection to DASH API #####
+dash.server = ${default.dash.server}
+dash.username = ${default.dash.username}
```

Comment:
Can you change these to match the new system's terminology? i.e. `client_id` instead of `username` and `client_secret` instead of `password`?